src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -118,7 +118,7 @@ def pull_helm_chart(registry_path, chart_export_path, kube_config, kube_context,
     chart_version = registry_path.split(':')[1]
 
     if new_path:
-        if (version.parse(chart_version) < version.parse("1.14.0")):
+        if '-' not in chart_version and (version.parse(chart_version) < version.parse("1.14.0")):
```

Comment:
Here too add a comment explaining why we are doing this. Something like "We want to do this check only for Stable versions, which will be of the pattern '1.x.x' versus other internal versions e.g. '1.x.x-dev'"